demos/python/sdk_wireless_camera_control/open_gopro/api/ble_commands.py

```diff
@@ -653,6 +653,7 @@ async def set_livestream_mode(
         minimum_bitrate: int,
         maximum_bitrate: int,
         starting_bitrate: int,
+        encode: bool,

```

Comment:
can you make this default to True and also move it to the last argument in order to maintain backwards compatability?

Reply:
Oh, gotcha, sorry was not following what was previously said. Fixed.